lib/driver.rb

```diff
@@ -43,5 +43,14 @@ def add_trip(trip)
 
       @trips << trip
     end
+
+    def driver_total_revenue
+      @trips.empty? ? 0 : (@trips.reduce(0) { |revenue, trip| revenue + trip.cost } - 1.65) * 0.8
```

Comment:
This is well written, but your math is a little off here.  You're subtracting `1.65` from the `total` of all the trips, rather than subtracting it from each trip.